### DISCUSSION TITLE
Strategy `prefix` does not forward to default locale from `/` if SSG

### DISCUSSION THREAD

**Discussion by frederikheld** [nuxt-modules/i18n#3164] [Q&A]
I'm generating static sites with `nuxi generate`. If i18n strategy is `prefix`, this will not generate an index.html in the root of `.output/public` but only in the sub directories for the language codes. This makes sense, the problem is that the user will not be forwarded to the default domain or the one that is detected in the browser if they open my url at `/`.

This is confusing, as in dev mode, forwarding works. It just does not work in SSG generated pages.

So my workaround is to use `prefix_and_default`, but then the default locale will not be forwarded to the language code.

Is there any way to use strategy `prefix` with forwarding to the respective language code for all languages in SSG mode?

**Comment by spreadpando** [comment]
I haven't found a workaround, there are several open issues to do with this an the lack of recent commits to this repo has me considering moving away from this nuxt plugin 🫤 
https://github.com/nuxt-modules/i18n/issues/3030

**Reply by frederikheld** [reply]
Thanks for your assessment on the situation. This Nuxt plugin indeed doesn't feel very good. There's only so many projects a single person can maintain before quality goes down the drain 🤷

 But vue-i18n is my go to plugin for i18n so it would be great if I could use it with Nuxt as well. Have you found a way to use it in Nuxt apps without the Nuxt plugin? If not, which alternative do you use?

**Comment by rayblair06** [comment]
It’s not the ideal solution, but as a workaround in our last project, we set up a redirect from the root to the default locale. This should fix the issue locally and the redirects should then be passed to your platform (we used Cloudflare).

```
export default defineNuxtConfig({
  routeRules: {
    "/": {
      redirect: "/en",
    },
  },
});
```

**Reply by frederikheld** [reply]
Thx! Does this work with browser locale detection?

**Reply by rayblair06** [reply]
I can't be sure and I don't have anyway to check (don't have access to those projects anymore).
